pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md

```diff
@@ -25,4 +25,4 @@ Follow these steps to upgrade an application from the DKP Dashboard:
 
 ### Upgrade with CLI
 
-<need to add instuctions>
+<!-- need to add instuctions and command -->
```

Comment:
```suggestion
<!-- Per Grace, for now we can use the following command (which needs to be changed before GA), but let's go ahead and document the procedure and then we can make a quick change to the cmd as necessary. -->

From the DKP CLI, use `DKP upgrade appdeployment` to upgrade a catalog application.

```

Reply:
@msdolbey @cneth - am I writing this accurately?

Reply:
@Colin1841 - I haven't had a chance to try this yet.   Will test soon.

Reply:
we are going to update `appdeployment` in the above commands to `catalogapp`